Misc/NEWS.d/next/Tools-Demos/2019-09-18-13-49-56.bpo-38080.Nbl7lF.rst

```diff
@@ -0,0 +1,2 @@
+Added getproxies to :const:`MAPPING` :mod:`Lib.lib2to3.fixes.fix_urllib`
```

Comment:
IMHO it would be a bit clearer if you say 
```suggestion
Added support to fix ``getproxies`` in the :mod:`lib2to3.fixes.fix_urllib`.
```

Reply:
@isidentical Done!, thanks!